documentation/src/main/asciidoc/public/reference/_backend-lucene.adoc

```diff
@@ -783,6 +783,40 @@ see the javadoc of `Similarity` and Lucene's source code.
 You can also find useful resources on the web, for example in Elasticsearch's documentation.
 ====
 
+[[backend-lucene-analysis-run-analysis]]
+=== Applying configured analyzers/normalizers to a string
```

Comment:
I get where you're coming from, but it feels like a shame that we end up duplicating content.

What do you think of doing this instead:

* Introduce a top-level section called "Explicit backend/index operations", perhaps just after "Searching" and before "Lucene backend"
* Introduce a subsection there called "Applying analyzers/normalizers to a string"

Then we could move more stuff there. For example, these look like they could belong in this new section:

* https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#mapper-orm-indexing-manual-largescale
* https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#backend-lucene-access-analyzers
* https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#backend-lucene-access-size
* https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#backend-lucene-access-index-reader
* https://docs.jboss.org/hibernate/stable/search/reference/en-US/html_single/#backend-elasticsearch-access-client

I've always thought it was weird that we put backend-specific predicates in the predicates section, but backend-specific operations in the backend sections... That would solve it.

We can totally do the moving of all these sections in a separate PR, though.

Reply:
ok I've made an attempt on moving these around 😃 🙈